docs/DebuggingNumericalError.md

```diff
@@ -11,7 +11,7 @@ intermediate results produced by these two backends layer by layer.
 ## Prerequisite
 - Set `ONNX_MLIR_HOME` environment variable to be the path to the HOME
   directory for onnx-mlir. The HOME directory for onnx-mlir refers to the
-  parent folder containing the `bin`, `lib`, etc sub-folders in which ONNX-MLIR
+  parent folder containing the `bin`, `lib`, etc sub-folders in which ONNX MLIR
```

Comment:
why "-" is removed?

Reply:
We had a mixture of referring to our project as ONNX-MLIR vs ONNX MLIR.  I chose the former, as it was use in the title of the README page. It cam be flipped if there is a consensus for the notation with the `-`.